setup.py

```diff
@@ -1,98 +1,29 @@
-import versioneer
-
 import os
-try:
-    import importlib.metadata as metadata
-    get_version = lambda x: metadata.version(x)
-    PkgNotFound = metadata.PackageNotFoundError
-    parse_version = lambda x: metadata.version(x)
-except ImportError:
-    import pkg_resources
-    get_version = lambda x: pkg_resources.get_distribution(x).version
-    PkgNotFound = pkg_resources.DistributionNotFound
-    parse_version = lambda x: pkg_resources.parse_version(x)
-
 from setuptools import setup, find_packages
+import versioneer
 
 
-def min_max(pkgs, pkg_name):
-    pkg = [p for p in pkgs if pkg_name in p][0]
-    minsign = '>=' if '>=' in pkg else '>'
-    maxsign = '<=' if '<=' in pkg else '<'
-    vmin = pkg.split(minsign)[1].split(',')[0]
-    vmax = pkg.split(maxsign)[-1]
-    return vmin, vmax
-
-
-def numpy_compat(required):
```

Comment:
Note: Ok so basically this was not doing anything because pip runs the build in isolation so sympy is never found and it always default to the `PkgNotFound` case.